AFQ/recognition/other_bundles.py

```diff
@@ -0,0 +1,67 @@
+import numpy as np
+
+import dipy.tracking.utils as dtu
+import dipy.tracking.streamline as dts
+
+from scipy.spatial.distance import cdist
+
+
+def clean_by_other_density_map(this_bundle_sls,
+                               other_bundle_sls,
+                               node_thresh,
+                               img):
+    """
+    Remove fibers that overlap with more than node_thresh nodes
+    """
+    other_bundle_density_map = dtu.density_map(
+        other_bundle_sls, np.eye(4), img.shape[:3])
+    fiber_probabilities = dts.values_from_volume(
+        other_bundle_density_map,
+        this_bundle_sls, np.eye(4))
+    cleaned_idx = np.zeros(len(this_bundle_sls), dtype=np.bool8)
+    for ii, fp in enumerate(fiber_probabilities):
+        cleaned_idx[ii] = np.sum(np.asarray(fp) >= 1) <= node_thresh
+    return cleaned_idx
+
+
+def clean_relative_to_other_core(core, this_fgarray, other_fgarray, affine):
+    """
+    Remove any fibers that are on the wrong side of the core
```

Comment:
Could you please add a full docstring? 

For this one in particular, it would be good to state the assumptions about orientation of the coordinates (RAS+?)

Reply:
There is no assumption about coordinates, as long as the affine is correct. see this line:
```
    if affine[core_axis, core_axis] < 0:
        core_direc = -core_direc
```